Scripts/build.swift

```diff
@@ -74,24 +86,27 @@ enum Platform: String, CaseIterable, CustomStringConvertible {
         switch self {
         case .iOS_12,
              .iOS_13,
+             .iOS_14,
              .tvOS_12,
              .tvOS_13,
+             .tvOS_14,
              .macOS_10_15:
             return true
 
         case .watchOS_5,
-             .watchOS_6:
+             .watchOS_6,
+             .watchOS_7:
             // watchOS does not support unit testing (yet?).
             return false
         }
     }
 
     var derivedDataPath: String {
-        ".build/derivedData/" + description
+        return ".build/derivedData/" + description
```

Comment:
In order to get access to watchOS 5 on GitHub Actions I needed to run this code on Xcode 10 which means I need `return` statements.